packages/react/src/Highlight/Highlight.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
+
+export interface HighlightProps extends PropsWithChildren<HTMLAttributes<HTMLDivElement>> {
+  color?: 'yellow' | 'orange' | 'magenta' | 'purple' | 'blue' | 'light-blue' | 'green' | 'dark-green'
```

Comment:
Let’s sort these as well:

```suggestion
  color?: 'blue' | 'dark-green' | 'green' | 'light-blue' | 'magenta' | 'orange' | 'purple' | 'yellow'
```